user-guide/mkdocs.yml

```diff
@@ -51,7 +51,7 @@ nav:
 
   - Simulation Applications: 
     - Overview: tools/simulation/overview.md
-    - ADCIRC: tools/simulation/adcirc.md
+    - ADCIRC: tools/simulation/adcirc/adcirc.md
```

Comment:
Why do this, to access new file, instead of updating old file? Do you have related questions I can answer?

Related comment: https://github.com/DesignSafe-CI/DS-User-Guide/pull/16/files#r1621304503

Reply:
Removed file, and keeping this as is